nemo_curator/utils/fuzzy_dedup_utils/shuffle_utils.py

```diff
@@ -169,6 +173,8 @@ def text_bytes_aware_shuffle(df, partition_on, text_column, num_workers=None):
     output_col = "_partitions"
 
     df = df.persist()
+    if len(df) == 0:
+        return None
```

Comment:
If we remove this conditional, then the test we added below fails
![Uploading image.png…]()

Reply:
Yup that testcase makes sense to me.